Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -73,7 +73,7 @@ public AwsUserProfilesPage(VisualElement container, StateManager stateManager)
             RefreshProfiles();
 
             container.Q<DropdownField>("UserProfilePageAccountNewProfileRegionDropdown").choices =
-                _stateManager.CoreApi.ListAvailableRegions().ToList();
+                _stateManager.CoreApi.ListAvailableRegions().OrderBy(str => str).ToList();
```

Comment:
Since the list of regions is hardcoded, I have a preference to re-order the list directly for now. That way this logic is simplified.